### ISSUE TITLE
Multiple notifications when an email is opened

### ISSUE THREAD

**Issue by CheechGe** [wavebox/waveboxapp#747]
* Wavebox Version: [e.g. 3.0.0]
Current Beta
..8 I think
* Operating System & Version: [e.g. Windows 10, Mac OSX 10.3.4, Ubuntu 17.10]
windows 10
* (Linux only) Install method: [e.g. Debian installer, Snap Package, Tar, Apt-Repo, etc...]
* Account type (if applicable): [e.g. Gmail, Office 365, Trello, etc...]
Gmail

**Expected & actual behavior**
Some information about what you expect to happen and what actually happens.
From some (not all) email recipients I now get notifications when the email is opened (nice feature).  However for one particular recipient I received about 4-5 notifications one after and other (within a 2-3 seconds at most)

**Steps to reproduce**
Are there any steps that we can take to reproduce the bug?
Not sure, this might have something to do with the the reciepient's email system and not a problem caused byeither GMail or Wavebox

**Is the bug persistent or intermittent?**
Do you only see it sometimes or does it always happen?
This was the first time that I noticed that happening since the problem with the alert sounds setting was corrected.  It may have happened before but was not as noticeable..

**Screenshots**
Screenshots often help us narrow down a bug more easily.
Not at this time.  There were 4 or 5 of them in the Notification window but they disappeared on their own.  If it happens again I will take a screen shot.

**Additional information**
The multiple notifications arrived so quickly it is unlikely that the email was in fact being opened multiple times.

**Comment by Thomas101** [comment]
I think the Gmail was incorrectly reporting duplicates last week. Are you still seeing this issue?

**Comment by CheechGe** [comment]
Hi Thomas

I was out of town for the last several days and not using wavebox so I
really can't tell you much.  The only thing that I can say is that the
multiple m=notification of when an email was read was from only one site.
I will send an email to thesite again in the next coupe of days and see
what happens.

Regards,

Frank

**Comment by CheechGe** [comment]
Hi Thomas:

I don't think that was the problem as it is happening again today and on a completely different site.  I responded to an email that was part of a website (not through my gmail account) and when the person open the email I received about 6 notifications one right after another.

Both occurrences were not from an email provider directly to another email provider.   

In the first occurrence I initiated the email from my gmail account (via WB) and sent it to a vendors support email address which I presume is part of a customized issue reporting software. In the second instance my email to the vendor was initiated from a Contact function on the vendor's website and I can only presume it was handled on the other end via some similar software.  

The notifications disappear from the notification panel before I can do a screen capture but they are exactly the same each time.  Neither site was connected through WB and were accessed using my default browser which is Vivaldi.

Hope that helps narrow things down.

Regards,

F

**Comment by CheechGe** [comment]
Below is a screen shot of multiple notifications.  I had just responded to a previous contact and when the addressee opened my email I received 3 notifications and possibly more as they disappear quickly.

![image](https://user-images.githubusercontent.com/3653639/43996260-cfc8ad46-9d8c-11e8-8c7b-2410e9d1040e.png)

**Comment by Thomas101** [comment]
@CheechGe do you have any extensions installed in Wavebox?

**Comment by CheechGe** [comment]
Yes, the one below, free edition

![image](https://user-images.githubusercontent.com/3653639/44036936-0f3d4ba8-9ee1-11e8-9186-89165adbdd76.png)

**Comment by Thomas101** [comment]
It might be worth disabling gmelius and giving it another try. Would help me narrow down the problem

**Comment by CheechGe** [comment]
Funny I was going to give that a try after you asked about whether there
were extensions installed  Will do.

**Comment by CheechGe** [comment]
I uninstalled gmelius as suggested and I did not get the duplicate read notifications, in fact I did not get any so I presume those were being done by gmelius.

One interesting thing that I hadn't noticed before is that Wavebox gives a notification for sent emails too, and I will explain how that can happen.

One of my PCs ( a desktop) has Wavebox connect to one of my (the primary one) email accounts.  On a different PC (a laptop) I use Thunderbird to connect to several email accounts, including the primary email account.  I do this because I did not want to install Thunderbird on my desktop but needed to get emails from my primary account.

In the case where I received a notification for a sent mail, I had responded to a previously sent mail on my primary account from Thunderbird.  When that email was sent I received a Wavebox notification from the sending of that email as if it was just received.  I presume that was because of the IMAP (or whatever one is being used) connection downloading the response email.  Apparently Wavebox (or whatever interface is being used) can not correctly distinguish this situation.  This is probably not a common configuration but is somewhat erroneous (but a very minor situation).
  
Also at this time I have not received any of those spurious Wavebox notifications reported in a different incident.

**Comment by Thomas101** [comment]
Okay thanks, I'll take a look to see if I can reproduce and get to the bottom of this

**Comment by CheechGe** [comment]
Yes it was their extension causing the duplicate email read notifications
(I uninstalled it last week).. Although I didn't check I suspect that
disabling is for the email read notifications feature and not to disable
the duplicate email notifications (which wouldn't make a lot of sense).
Anyhow, the read notifications was one of the reasons I installed it but
the duplicates are more annoying than getting them. I suppose I could
report it and maybe they will fix it.

This incident can be closed. Thanks.
